demo/deploy-mysql.sh

```diff
@@ -46,10 +46,8 @@ spec:
         app: mysql
     spec:
       serviceAccountName: mysql
-      nodeSelector:
-        kubernetes.io/os: linux
```

Comment:
In this case, keeping the nodeSelector for linux is probably a good idea since it doesn't look like this image works on Windows.

Reply:
Ah, good call, I did a sweeping pass. Totally agree.